Assets/Package/Runtime/Tooltip/Factories/StandardTooltipFactory.cs

```diff
@@ -10,7 +10,7 @@ namespace BroWar.UI.Tooltip.Factories
     [Serializable]
     public class StandardTooltipFactory : ITooltipFactory
     {
-        private readonly Dictionary<int, NativeObjectPool<TooltipBehaviour>> poolsByIds
+        private readonly IDictionary<int, NativeObjectPool<TooltipBehaviour>> poolsByIds
```

Comment:
IEnumerable od razu XD

Reply:
A tak na temat to może warto się zastanowić nad implementacją MultiObjectPool w Corze, która nam będzie wrapować takie przypadki? (Podobnie będzie w Workerach)

Reply:
Muszę się jeszcze zastanowić bo nie do końca jestem fanem tej implementacji